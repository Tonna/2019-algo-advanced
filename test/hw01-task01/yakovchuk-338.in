25
1 1 1 3 3 1 6 1 4 7 10 6 5 11 12 6 3 3 1 6 17 13 21 2
5
5 3
20 3
15 6
8 10
1 5
