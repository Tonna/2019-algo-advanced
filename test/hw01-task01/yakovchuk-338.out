5
22
26
28
7
